packages/suite-web-landing/components/Download/index.tsx

```diff
@@ -158,15 +158,17 @@ const Index = ({ pathToApp }: { pathToApp: string }) => {
                 ),
                 noPadding: true,
                 callback: () => setPlatform(item.platform),
-            },
-        ],
-    }));
+            })),
+        },
+    ];
 
     useEffect(() => {
         const defaultPlatform = getPlatform(window.navigator);
         setPlatform(defaultPlatform);
     }, []);
 
+    if (platform === null) return null;
```

Comment:
Would be really nice to avoid "jumping" of the content

https://user-images.githubusercontent.com/3729633/140305799-11477437-8a88-483d-95b4-40eceea055cc.mov

What about to render the button, just no icon and no links?

Reply:
fixed and squashed